robot_driver:
  ros__parameters:
    driver_type: "mujoco"   # 可选: "mujoco" 或 "roshw"